markdown/blog/2025-singapore-conf-summary.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Bringing Asynchronous APIs to the Forefront at APIDays Singapore"
+date: 2025-04-28T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+  - Conference Update
+cover: /img/posts/marketing-images/singapore-conf/singapore-header.webp
+authors:
+  - name: Atinuke Oluwabamikemi Kayode
+    photo: /img/avatars/bami.webp
+    link: https://www.linkedin.com/in/atinuke-oluwabamikemi-kayode-5b838b1b7/
+    byline: AsyncAPI Community Marketing Specialist
+excerpt: "AsyncAPI kicked off its 2025 conference with a packed track at APIDays Singapore—the first ever AsyncAPI presence at an APIDays event in Asia."
+---
+
+## Hello, Asia-Pacific
+
+In the early days of AsyncAPI, it all began with a question:
+**“What if developers could build event-driven systems as seamlessly as they build RESTful APIs?”**
+
+What started as a side project has become an industry standard, backed by a global community of engineers, architects, documentarians, and educators from every corner of the world.
+
+Over the years, we’ve shown up: at conferences, in GitHub issues and Slack threads—building not just tools, but a community. And in April 2025, we showed up somewhere new: **Singapore.**
+
+<Figure
+  src="/img/posts/marketing-images/singapore-conf/community-marketing-specialist.webp"
+  caption="Booth Moment With Our Marketing Specialist."
+  className="text-center"
+/>
+
+## First Time in Southeast Asia
+
+There’s something special about firsts. The first version of a spec. The first contributor. The first working group. The first time you explain what “event-driven” means to someone and see the lightbulb go off. This made us ecstatic to kick off the AsyncAPI conference in Singapore.
+On April 15–16, the AsyncAPI Conference was part of the [APIDays Singapore](https://www.apidays.global/singapore/) track. The event, themed "Where APIs Meet AI: Building Tomorrow's Intelligent Ecosystems," providing an excellent platform to introduce AsyncAPI to the vibrant Asia-Pacific developer community.​
+
+<Figure
+  src="/img/posts/marketing-images/singapore-conf/asyncapi-x-community.webp"
+  caption="Ankit Interacting With The Audience at The Booth."
+  className="text-center"
+/>
+
+Our dedicated AsyncAPI track was met with overwhelming interest. Sessions were filled to capacity, with attendees eager to explore the nuances of asynchronous APIs and event-driven architectures. The enthusiasm underscored the growing relevance of AsyncAPI in modern API development. And that mattered to us—not just because it showed interest, but because it proved that the problems we’re solving resonate globally.
```

Comment:
```suggestion
The AsyncAPI track had an overwhelming interest. Sessions were filled with attendees eager to explore the nuances of asynchronous APIs and event-driven architectures. The enthusiasm underscored the growing relevance of AsyncAPI in modern API development. And that mattered to us because it showed interest and proved that the problems we’re solving resonate globally.
```